pkg/metrics/metrics.go

```diff
@@ -216,7 +216,7 @@ The label 'underlying_cause' can have the following values:
 			Name:      "ready_wait_time_seconds",
 			Help:      "The time between a workload was created or requeued until ready, per 'cluster_queue'",
 			Buckets:   generateExponentialBuckets(14),
-		}, []string{"cluster_queue"},
+		}, []string{"cluster_queue", "workload_priority_class"},
```

Comment:
Why do we want to sore only WorkloadPriorityClass name?
Indeed, the Pod PriorityClass is used for Kueue scheduling and preemptions.

Shouldn't we add `priority_class` and `priority_calss_source` labels?

Reply:
Actually I thought about this more and I agree with the priority_class direction. 

It is not ticketed yet but we are considering another priority mechanism, where the priority source would be a configMap or another CrD. This way some automation could be adjusting all priorities for all workloads by a single API request. This is a separate feature which is not for sure yet going to be worked on, but it shows we may have new priority mechanisms.

So to make this more easy to extend I support rename to priority_clasass.

Still it this PR can stay with old name. We need another rename PR anyway. as for the priority_class_source it may be added later imo

Reply:
For ClusterQueue, we only have one PR: https://github.com/kubernetes-sigs/kueue/pull/6886.